tests/integration/select-component-render-test.js

```diff
@@ -507,7 +507,7 @@ test('Selected option is visible when dropdown is opened', function(assert) {
   this.render();
   // The highlighted property is set when the user hovers over the select field
   // Lets programatically set it after rendering to emulate that behavior.
-  andThen(() => select.set('highlighted', selection));
+  andThen(() => { select.set('highlighted', selection); });
```

Comment:
that does not seem related, does it?

Reply:
The codemod picks it up because it's passed a return value of the callback.

Reply:
I manually fixed this, otherwise it would have been changed to

```js
andThen(() => (select.set('highlighted', selection), select));
```